### ISSUE TITLE
Can't modify sensor group

### ISSUE THREAD

**Issue by clukasek** [nielsfaber/alarmo#860]
### Alarmo Version

v1.9.10

### HA Version

2023.10.3

### Bug description

When trying to modify a sensor group (e.g. two motion sensors combined), either to add/remove a sensor, or to change the time-out amount, I get the following error.

![image](https://github.com/nielsfaber/alarmo/assets/144564743/984c7cd7-376d-449a-ad7b-d738fdcb1bfe)

### Steps to reproduce

Sensors --> setup groups --> any change to an existing group

**Comment by nielsfaber** [comment]
I'm trying to reproduce this issue, but I couldn't.
This is my sensor group:
![image](https://github.com/nielsfaber/alarmo/assets/16861605/16c6e6e2-121f-4a0c-845b-b8b99d449802)

If I add a third sensor and click save, or save without changing anything, this doesn't give me an error.
What are you doing/seeing different than me?
Do you see the sensor entities mentioned in the error being selected prior to saving?

**Comment by clukasek** [comment]
Yes, they're selected as shown here. Changing anything, or not making changes and hitting save, results in the error in original post.
![Screenshot_20240101-090240](https://github.com/nielsfaber/alarmo/assets/144564743/41349514-40d4-4489-9f3a-3a5e5a209571)

**Comment by clukasek** [comment]
Didn't mean to "close issue"

**Comment by VNRARA** [comment]
Groups sensors are so buggy in general. Idk wtf is even happing.

**Comment by PowerBat80** [comment]
I have written some more info about this bug in this comment:
https://github.com/nielsfaber/alarmo/issues/868#issuecomment-1888104236
Regards

**Comment by nielsfaber** [comment]
This should be fixed by updating to the v1.9.14 release. 
My apologies for the issue.

**Comment by VNRARA** [comment]
Thanks for looking into this. Will check again soon.

**Comment by VNRARA** [comment]
Seems to be working fine now :)
